snapshot-labs/snapshot
# fix: Update shutter logo

### Summary
Similar changes with https://github.com/snapshot-labs/sx-monorepo/pull/558
| Before    | After |
| -------- | ------- |
| <img width="355" alt="Untitled 6" src="https://github.com/user-attachments/assets/c8d85d23-da20-4891-b1c6-e33830c4db7d"> | <img width="340" alt="Untitled 5" src="https://github.com/user-attachments/assets/bc2085c5-8d1f-4676-8f2b-6197e649d2c0"> |



### How to test

1. go to http://localhost:8080/#/shutterdao0x36.eth/proposal/0x4bb5cf27998b7371d1c69f7d2d0e433d601bcb46e2a2cdd38108a743c432fa35
2. you should see updated logo
